tempesta_fw/t/functional/long_body/test_long_request.py

```diff
@@ -0,0 +1,68 @@
+""" Testing for long body in request """
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+import unittest
+import body_generator
+import os
+
+from testers import stress
+from helpers import tf_cfg, control, tempesta, remote, wrk
+
+class RequestTestBase(stress.StressTest):
+    """ Test long request """
+    config = "cache 0;\n"
+    root = "/tmp/long_body/"
+    script = None
+    scriptdir = root + "scripts/"
+    wrk = None
+    clients = []
+    generator = None
+
+    def create_clients_with_body(self, length):
+        """ Create wrk client with long request body """
+        self.generator = wrk.ScriptGenerator()
+        self.generator.set_body(body_generator.generate_body(length))
+        if not os.path.exists(self.root):
```

Comment:
Use `remote.node.mkdir()` instead: first it always care about on which node should it make the directory, secondly it already throws an exception for test suit on errors.

Reply:
wrk scripts copy code fixed